src/components/_modal/pages/deleteOrder/DeleteOrder.page.js

```diff
@@ -52,7 +52,7 @@ function DeleteOrder({ context, modal }) {
           className={styles.confirmationText}
           dataCy="material-title"
         >
-          {title}
+          {`"${title}"`}
```

Comment:
Ellers kan du skrive "{title}"

Reply:
det er eslint ikke glad for, men jeg kan escape dem